### ISSUE TITLE
option to filter clustered snps

### ISSUE THREAD

**Issue by ShannonDaddy** [Illumina/Pisces#30]
Hi, is there an option for Pisces to filter clustered snps, usually they are false postives.

**Comment by tamsen** [comment]
We don't have it in Pisces. Gemini might have an option to filter messy reads from the bam before they get to Pisces (which IMO is the logical place for that). If we don't yet, its a good idea to add. :-) Let me check with Gwenn.

**Comment by gwennberry** [comment]
Yes, Gemini does have an option that may be useful for this scenario (filtering reads with high mismatch count if they look suspicious) - it will be available in the next release. Stay tuned for that! Also, in some cases these clustered SNVs are actually misaligned indels, in which case the indel realignment functionality in the available version of Gemini will help.
